src/test_solver.jl

```diff
@@ -0,0 +1,63 @@
+import Base.Test: @test, @test_approx_eq_eps
+
+const testset = [
```

Comment:
As suggested in #20 and #21, maybe have the ode and y0 separate to mix and match odes and types.  But this can also be done in a later commit/PR.